From b804a4fcaa829eca1ed254e569f626eef9c64bcd Mon Sep 17 00:00:00 2001
From: Jan Kratochvil <jan.kratochvil@redhat.com>
Date: Wed, 9 May 2012 19:29:25 +0000
Subject: [PATCH] gdb/doc/
 	* gdb.texinfo (Separate Debug Files, Auto-loading safe path): Replace
 	directory separator by path separator.

---
 gdb/doc/ChangeLog   |    3 +++
 gdb/doc/gdb.texinfo |    8 ++++----
 2 files changed, 7 insertions(+), 4 deletions(-)

Index: gdb-7.2/gdb/doc/gdb.texinfo
===================================================================
--- gdb-7.2.orig/gdb/doc/gdb.texinfo
+++ gdb-7.2/gdb/doc/gdb.texinfo
@@ -15186,8 +15186,8 @@ name @value{GDBN} is currently using.
 @kindex set debug-file-directory
 @item set debug-file-directory @var{directories}
 Set the directories which @value{GDBN} searches for separate debugging
-information files to @var{directory}.  Multiple directory components can be set
-concatenating them by a directory separator.
+information files to @var{directory}.  Multiple path components can be set
+concatenating them by a path separator.
 
 @kindex show debug-file-directory
 @item show debug-file-directory
@@ -19988,7 +19988,7 @@ loading and execution of scripts.  You c
 If you omit @var{directories}, @samp{auto-load safe-path} will be reset to
 its default value as specified during @value{GDBN} compilation.
 
-The list of directories uses directory separator (@samp{:} on GNU and Unix
+The list of directories uses path separator (@samp{:} on GNU and Unix
 systems, @samp{;} on MS-Windows and MS-DOS) to separate directories, similarly
 to the @env{PATH} environment variable.
 
@@ -20003,7 +20003,7 @@ scripts.
 @item add-auto-load-safe-path
 Add an entry (or list of entries) the list of directories trusted for automatic
 loading and execution of scripts.  Multiple entries may be delimited by the
-host platform directory separator in use.
+host platform path separator in use.
 @end table
 
 Setting this variable to @file{/} disables this security protection.
